lldb/test/Shell/Commands/command-statistics-dump.test

```diff
@@ -0,0 +1,31 @@
+# RUN: %clang_host -g %S/Inputs/main.c -o %t-main.exe
+
+# When we enable symbol preload and dump stats there should be a non-zero
+# time for parsing symbol tables for the main module.
```

Comment:
Can you add top level comment explicitly point out that the test tries to only create the main target module without running or loading dependency to avoid any runtime plugin global symbol lookup. This is a key assumption.

Reply:
Added the comment, thanks!